docs/installation/index.md

```diff
@@ -1,8 +1,6 @@
-<!--dj-proj-start-->
+???+ tip "Learn Django before reading these docs!"
```

Comment:
This sounds a bit harsh (like "just get good" in gaming). Maybe something like:

```suggestion
???+ tip "We recommend having basic knowledge of [Django](https://docs.djangoproject.com) before using Django IDOM"
```

Reply:
I've changed the verbiage to "Learning Django first is recommended!".

Let me know if this works.